lib/translator/translation.cc

```diff
@@ -129,6 +130,30 @@ BeginResponse Translation::Begin(Span<const uint8_t> scsi_cmd,
       pipeline_status_ = VerifyToNvme(scsi_cmd_no_op, nvme_wrappers_[0]);
       nvme_cmd_count_ = 1;
       break;
+    case scsi::OpCode::kWrite6:
+      pipeline_status_ =
+          Write6ToNvme(scsi_cmd, nvme_wrappers_[0], allocations_[0], nsid,
+                       kPageSize, kLbaSize, buffer_out);
+      nvme_cmd_count_ = 1;
+      break;
+    case scsi::OpCode::kWrite10:
+      pipeline_status_ =
+          Write10ToNvme(scsi_cmd, nvme_wrappers_[0], allocations_[0], nsid,
+                        kPageSize, kLbaSize, buffer_out);
+      nvme_cmd_count_ = 1;
+      break;
+    case scsi::OpCode::kWrite12:
+      pipeline_status_ =
+          Write12ToNvme(scsi_cmd, nvme_wrappers_[0], allocations_[0], nsid,
+                        kPageSize, kLbaSize, buffer_out);
+      nvme_cmd_count_ = 1;
+      break;
+    case scsi::OpCode::kWrite16:
+      pipeline_status_ =
+          Write16ToNvme(scsi_cmd, nvme_wrappers_[0], allocations_[0], nsid,
+                        kPageSize, kLbaSize, buffer_out);
+      nvme_cmd_count_ = 1;
+      break;
     default:
       DebugLog("Bad OpCode: %#x", static_cast<uint8_t>(opc));
       pipeline_status_ = StatusCode::kFailure;
```

Comment:
Even though there isn't a `WriteToScsi`, in Translation::Complete, we still have to set the pipeline_status_ to kSuccess for the write commands.